Safewaan/HonkedIn
# users can edit their forums via the my forums page

## Trello ticket link

[As a user, I want to be able to edit the displayed information for my forums.](https://trello.com/c/g24mrfWt )



## Implementation description
* Users can now edit their forums through the My Forums page
* They can open an edit forum window by clicking a button imbedded in each listed forum
* These changes are reflected in the database
* Added links to individual forum pages on the My Forums page
* ADDITIONALLY - added unit test file for Forums page


## Steps to test
1. On the My Forums page, make sure you can see the "EDIT FORUM" button 
2. Verify that clicking on the button opens a pre-filled editing form with the existing field values 
3. Verify that you can replace the values for either or both fields
4. Verify that if you click "edit forum" with a field left empty, that it gives an error and does not accept the change to the forum
5. Check that any changes to the fields are saved to the db and displayed on the My Forums and Forums page after the page resets once you click the "edit forum" button



## What should reviewers focus on?
* Make sure that the error flagging works
* Make sure that the edit forum functionality works on both the front and back end 
* Please also check that my unit tests all pass
